test/libsolidity/smtCheckerTests/types/array_branches_3d.sol

```diff
@@ -3,13 +3,22 @@ pragma experimental SMTChecker;
 contract C
 {
 	uint[][][] c;
+	constructor() {
+		c.push();
+		c[0].push();
+		c[0][0].push();
+	}
 	function f(bool b) public {
 		c[0][0][0] = 0;
 		if (b)
 			c[0][0][0] = 1;
-		else
-			c[0][0][0] = 2;
-		assert(c[0][0][0] > 0);
+		assert(c[0][0][0] < 2);
 	}
 }
 // ----
+// Warning 6368: (157-164): CHC: Out of bounds access might happen here.
+// Warning 6368: (157-167): CHC: Out of bounds access might happen here.
+// Warning 6368: (185-192): CHC: Out of bounds access might happen here.
+// Warning 6368: (185-195): CHC: Out of bounds access might happen here.
+// Warning 6368: (210-217): CHC: Out of bounds access might happen here.
+// Warning 6368: (210-220): CHC: Out of bounds access might happen here.
```

Comment:
Is it really happening?
For me looks like it shouldn't report. If it reports because of a bug, we should mention in comments somewhere. I see on other similar places as well, so will not comment there for now

Reply:
Same with `might happen`, solver couldn't prove it